scripts/gen_help_html.lua

```diff
@@ -512,6 +532,8 @@ local function visit_node(root, level, lang_tree, headings, opt, stats)
     end
     local el = in_heading and 'span' or 'code'
     local s = ('%s<a name="%s"></a><%s class="%s">%s</%s>'):format(ws(), url_encode(tagname), el, cssclass, trimmed, el)
+    s = fix_tab_after_conceal(s, node_text(root:next_sibling()))
```

Comment:
In all cases, this also only is needed if `opt.old` is true

Reply:
Yeah I was being a little lazy there. I think with or without the two-space insertion it still generates the same rendered output but then it's not necessary to add the two spaces for new-style ones then. Added the check. 

Just to confirm, after my change, now we can see that the new style docs weren't changed:

```
➜ ychin ~/Dev/external/neovim/target% diff dir/api.html dir_clean/api.html 
5064c5064
<         Generated at 2022-10-19 17:23 from <code><a href="https://github.com/neovim/neovim/commit/?">?</a></code>
---
>         Generated at 2022-10-19 17:22 from <code><a href="https://github.com/neovim/neovim/commit/?">?</a></code>
➜ ychin ~/Dev/external/neovim/target% diff dir/nvim.html dir_clean/nvim.html
169c169
<         Generated at 2022-10-19 17:23 from <code><a href="https://github.com/neovim/neovim/commit/?">?</a></code>
---
>         Generated at 2022-10-19 17:22 from <code><a href="https://github.com/neovim/neovim/commit/?">?</a></code>
```